src/dropdown/dropdown.scss

```diff
@@ -221,17 +221,10 @@ $height-small: 30px !default;
       height: auto;
 
       .Select-multi-value-wrapper {
-        min-height: $height;
         height: auto;
```

Comment:
And the input: if you make it `searchable` you don't see what you type (if there are no value)

I'm not 100% sure, but I think you can fix it using the `.has-value` class and apply the min-height conditionally